src/pydargs/__init__.py

```diff
@@ -135,6 +132,18 @@ def _create_parser(tp: Type[Dataclass]) -> ArgumentParser:
     return parser
 
 
+Fct = TypeVar("Fct")
+
+
+def rename(name: str):
+    def _rename(f: Fct) -> Fct:
+        setattr(f, "__name__", name)
+        return f
+
+    return _rename
+
+
+@rename("bool")
```

Comment:
What would go wrong if we didn't set it to "bool"? Or is it just for convenience for the user to understand what goes wrong if a bool argument is misspecified?

Reply:
This test would break;

```python
assert "error: argument --b: invalid bool value:" in captured.err
```


It would instead say `invalid _parse_bool value` as it uses the `__name__` of the type caster for the message